Dockerfile

```diff
@@ -0,0 +1,14 @@
+# For travis
+FROM ubuntu:16.04
+RUN apt-get update
+RUN apt-get install -y python wget cmake
+RUN wget https://s3.amazonaws.com/mozilla-games/emscripten/releases/emsdk-portable.tar.gz
+RUN tar -xvf emsdk-portable.tar.gz
+RUN cd emsdk-portable && ./emsdk update
+RUN cd emsdk-portable && ./emsdk install latest
+RUN cd emsdk-portable && ./emsdk activate latest
+RUN ls
+RUN ./emcc # first use
+RUN ./emcc -v # check s tuff
+RUN python tests/runner.py test_hello_world
```

Comment:
Right now this Docker file will consume 11 layers, it will be better to make it look like: 
```
RUN apt-get update \
&&    apt-get install -y python wget cmake \
&&    wget https://s3.amazonaws.com/mozilla-games/emscripten/releases/emsdk-portable.tar.gz \
&&    tar -xvf emsdk-portable.tar.gz \
&&    cd emsdk-portable \
&&    ./emsdk update \
&&    ./emsdk install latest \
&&    ./emsdk activate latest \
&&    ls \
&&    ./emcc \
&&    ./emcc -v \
&&    \
&&    python tests/runner.py test_hello_world
```
It's important, because each `RUN`  creates a snapshot. Especially step where you install latest will expand size of the image